ruby/metaparticle/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
I'd like to have a single LICENSE at the top, rather than a lower-level one in a language implemenation. Hope that makes sense...